.github/workflows/build_and_test.yml

```diff
@@ -80,6 +80,7 @@ jobs:
       if:  (matrix.os == 'ubuntu-22.04') && (github.event_name == 'pull_request' || github.event_name == 'schedule' )
       run: |
         cd docs && bash ./install.sh
+        poetry run pip install ../.
```

Comment:
Not sure if this is how I should be installing the pytket-qiskit version in this workflow. 

Would it be better to install a `.whl` directly?

Reply:
I think there should be a wheel available to install at this point.
Could you try something like below?
```
run: for w in `find wheelhouse/ -type f -name "*.whl"` ; do poetry install $w ; done
```
Or if that is not working
```
run: for w in `find wheelhouse/ -type f -name "*.whl"` ; do poetry run pip install $w ; done
```